.pre-commit-config.yaml

```diff
@@ -41,6 +41,16 @@ repos:
     files: \.(yaml|yml)$
     types: [file, yaml]
     entry: yamllint --strict
+# - repo: https://github.com/asottile/seed-isort-config
+#   rev: v2.2.0
+#   hooks:
+#   - id: seed-isort-config
+#     args: [--application-directories=lib]
```

Comment:
why?

Reply:
thanks, i removed it. FYI, I tried to use it but realised it is buggy and adds undesired complexity. Our deps list are quite stable and we
can maintain the list ourselves.